docs/mine/lotus/miner-wallets.md

```diff
@@ -1,145 +1,149 @@
+
+   > name       ID      key           use    balance
+   > owner      t01111  f3abcd...  other  300 FIL
+   > worker     t01111  f3abcd...  other  300 FIL
+   > control-0  t02222  f3defg...  post   100 FIL
+   ```
 
-You can repeat this procedure to add additional addresses.
+Repeat this procedure to add additional addresses.
 
 ## Managing balances
 
-To get the balances associated with your miner wallet, run:
+Get the balances associated with a miner wallet by calling `info`:
 
 ```bash
-$ lotus-miner info
-Miner: t01000
-Sector Size: 2 KiB
-Byte Power:   100 KiB / 100 KiB (100.0000%)
-Actual Power: 1e+03 Ki / 1e+03 Ki (100.0000%)
-	Committed: 100 KiB
-	Proving: 100 KiB
-Below minimum power threshold, no blocks will be won
-Deals: 0, 0 B
-	Active: 0, 0 B (Verified: 0, 0 B)
-
-Miner Balance: 10582.321501530685596531 FIL
-	PreCommit:   0.000000286878768791 FIL
-	Pledge:      0.00002980232192 FIL
-	Locked:      10582.321420164834231291 FIL
-	Available:   0.000051276650676449 FIL
-Worker Balance: 49999999.999834359275302423 FIL
-Market (Escrow):  0 FIL
-Market (Locked):  0 FIL
+lotus-miner info
+
+> Miner: t01000
+> Sector Size: 2 KiB
+> Byte Power:   100 KiB / 100 KiB (100.0000%)
+> Actual Power: 1e+03 Ki / 1e+03 Ki (100.0000%)
+>   Committed: 100 KiB
+>   Proving: 100 KiB
+> Below minimum power threshold, no blocks will be won
+> Deals: 0, 0 B
+>   Active: 0, 0 B (Verified: 0, 0 B)
+>
+> Miner Balance: 10582.321501530685596531 FIL
+>   PreCommit:   0.000000286878768791 FIL
+>   Pledge:      0.00002980232192 FIL
+>   Locked:      10582.321420164834231291 FIL
+>   Available:   0.000051276650676449 FIL
+> Worker Balance: 49999999.999834359275302423 FIL
+> Market (Escrow):  0 FIL
+> Market (Locked):  0 FIL
 ```
 
-Here, miner id is `t01000`, and it has total balance of `10582.321501530685596531 FIL` with an available balance of `0.000051276650676449 FIL` that can be used as collateral/pay for the pledge. The worker balance is `49999999.999834359275302423 FIL`.
+In this example, the miner ID is `t01000`, it has a total balance of `10582.321501530685596531 FIL`, and an available balance of `0.000051276650676449 FIL` that can be used as collateral or to pay for the pledge. The worker balance is `49999999.999834359275302423 FIL`.
 
 ## Withdrawing funds from the Miner actor
 
-The funds in the _Miner actor_ can be withdrawn to the _owner address_ using:
+Transfer funds from the Miner actor address to the owner address by calling `actor withdraw`:
 
 ```bash
 lotus-miner actor withdraw <amount>
 ```
 
 ::: tip
-Note that the owner address will need to be available in the Lotus Node and have enough funds to pay for the gas for this transaction. Thus, cold wallets will need to be temporally imported for the operation to succeed.
+The owner's address will need to be available in the Lotus node and have enough funds to pay for the gas for this transaction. Cold wallets will need to be temporally imported for the operation to succeed.
```

Comment:
```suggestion
The owner's address will need to be available in the Lotus node and have enough funds to pay for the gas for this transaction. Cold addresses will need to be temporally imported for the operation to succeed.
```

(or addresses in cold wallets. But what is imported is the address, not the wallet).